### ISSUE TITLE
Enhancement: twiss to Pandas

### ISSUE THREAD

**Issue by pbelange** [xsuite/xsuite#174]
For tabular data like twiss, it would be useful to have a `to_pandas` method, which is not straightforward for now since `tracker.twiss()` returns a dictionary of data which are not all tabular. In my code I use:
```python
df_columns = ['name', 's', 'x', 'px', 'y', 'py', 'zeta', 'delta', 'ptau', 'betx', 'bety', 'alfx', 'alfy', 'gamx', 'gamy', 'dx', 'dpx', 'dy', 'dpy', 'mux', 'muy', 'qx', 'qy', 'qs', 'dqx', 'dqy']

xtwiss  = xt.Tracker(line=line).twiss()
twiss   = pd.DataFrame(xtwiss,columns = df_columns)
twiss.set_index('name',inplace=True)
```

**Comment by giadarol** [comment]
The last steps will have to be left to the user since in some cases names are repeated

**Comment by giadarol** [comment]
Implemented in xtrack 0.20.0
